doc/texdoc.1.md

```diff
@@ -49,8 +49,10 @@ Alternatively, perform the given ACTION and exit.
   Print the version number.
 * `-f`, `--files`:
   Print the list of configuration files used.
-* `--just-view` FILE:
-  Display FILE, given with full path (no searching).
+* `--just-view`:
```

Comment:
I think we have to keep `FILE` here since `--just-view` is in the `action` section.

https://github.com/TeX-Live/texdoc/blob/b8f7ebd9eb14ad57d9d3f04e21fecaad349f6cf5/script/texdoclib-const.tlu#L31-L32

https://github.com/TeX-Live/texdoc/blob/b8f7ebd9eb14ad57d9d3f04e21fecaad349f6cf5/script/texdoclib-const.tlu#L54-L58